modules/mono/glue/GodotSharp/GodotSharp/Core/Transform2D.cs

```diff
@@ -552,11 +527,11 @@ public Transform2D(real_t rotation, Vector2 origin)
             Vector2 to2 = new Vector2(rect.Position.x + rect.Size.x, rect.Position.y + rect.Size.y) * transform;
             Vector2 to3 = new Vector2(rect.Position.x + rect.Size.x, rect.Position.y) * transform;
 
-            return new Rect2(pos, rect.Size).Expand(to1).Expand(to2).Expand(to3);
+            return new Rect2(pos, new Vector2()).Expand(to1).Expand(to2).Expand(to3);
```

Comment:
This is a separate bug fix. It looks good, but it's questionable to sneak this in with Xform refactor.

Reply:
You are right, I thought since I was modifying all the Xform methods and I noticed this difference with the C++ implementation it might make sense to include it rather than create a separate PR but probably should extract it.

Reply:
I'm fine with it being in this PR, or another, but it should at least be mentioned in the PR description.